.travis.yml

```diff
@@ -0,0 +1,16 @@
+language: cpp
+sudo: required
+
+services: docker
+
+before_install:
+ - docker pull ubuntu
+ - docker run --cidfile /tmp/id -it -d ubuntu /bin/bash
+ - tar cf csd.tar * && docker cp csd.tar $(cat /tmp/id):/tmp/
+ - docker exec -i $(cat /tmp/id) /bin/bash -c "apt-get update -qq && apt-get install -y -q libavahi-glib-dev libavahi-client-dev libavahi-core-dev libgstreamer1.0-dev libgstrtspserver-1.0-dev libgstreamer-plugins-base1.0-dev build-essential autoconf libtool wget"
+   # newer libgstrtspserver packages
+ - docker exec -i $(cat /tmp/id) /bin/bash -c "wget http://security.ubuntu.com/ubuntu/pool/universe/g/gst-rtsp-server1.0/libgstrtspserver-1.0-0_1.8.3-1_amd64.deb && wget http://security.ubuntu.com/ubuntu/pool/universe/g/gst-rtsp-server1.0/libgstrtspserver-1.0-dev_1.8.3-1_amd64.deb && wget http://security.ubuntu.com/ubuntu/pool/universe/g/gst-rtsp-server1.0/gir1.2-gst-rtsp-server-1.0_1.8.3-1_amd64.deb && dpkg -i *.deb"
```

Comment:
This was using dpkg -i. Yep, universe could be enabled instead of it. 
Notice this commit is deprecated, though. Moving to a docker image with these things already installed, which makes the whole process faster instead of setting the environment over and over again.